src/main/java/org/cryptomator/launcher/SupportedLanguages.java

```diff
@@ -7,17 +7,21 @@
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
+import java.text.Collator;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Locale;
 
 @Singleton
 public class SupportedLanguages {
 
 	private static final Logger LOG = LoggerFactory.getLogger(SupportedLanguages.class);
-	// these are BCP 47 language codes, not ISO. Note the "-" instead of the "_":
-	public static final List<String> LANGUAGAE_TAGS = List.of("en", "ar", "be", "bn", "bs", "ca", "cs", "da", "de", "el", "es", "fil", "fa", "fr", "gl", "he", //
+	// these are BCP 47 language codes, not ISO. Note the "-" instead of the "_".
+	// "en" is not part of this list - it is always inserted at the top.
+	public static final List<String> LANGUAGE_TAGS = Arrays.asList("ar", "be", "bn", "bs", "ca", "cs", "da", "de", "el", "es", "fil", "fa", "fr", "gl", "he", //
```

Comment:
A list returned by `Arrays.asList(...)` is modifable. I woudl prefer, that we encapsulate the list and offer other classes only an unmodifable view.

Can you refactor the code, such that the list is private and we have a getter method returning an unmodifiable list? (e.g. by [Collections.unmodifiableList(...)](https://docs.oracle.com/en/java/javase/19/docs/api/java.base/java/util/Collections.html#unmodifiableList(java.util.List)))

Reply:
Been there, done that :)